test/expand.coffee

```diff
@@ -1,11 +1,10 @@
-{ PinejsClientTest } = require './test'
+{ test } = require './test'
 
 testExpand = (input, output) ->
 	resource = 'test'
-	url = '/' + resource + '?$expand=' + output
+	url = resource + '?$expand=' + output
```

Comment:
So we now just compile the request directly, which results in something like `resource?$filter=a eq 'b'` without the apiPrefix (which is added on an actually http request)